admin/common_auth/admin.py

```diff
@@ -0,0 +1,63 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import Permission
+from .models import MyUser
+
+from django.contrib.auth.forms import PasswordResetForm, UserCreationForm
+
+class PermissionAdmin(admin.ModelAdmin):
+    search_fields = ['name', 'codename']
+
+class CustomUserRegistrationForm(UserCreationForm):
+    class Meta:
+            model = MyUser
+            fields = ['password', 'first_name', 'last_name', 'email', 'is_active', 'is_staff',
+            'is_superuser', 'groups', 'user_permissions', 'last_login', ]
+    def __init__(self, *args, **kwargs):
+        super(CustomUserRegistrationForm, self).__init__(*args, **kwargs)
+        self.fields['first_name'].required = True
+        self.fields['last_name'].required = True
+
+class CustomUserAdmin(UserAdmin):
+    add_form = CustomUserRegistrationForm
+    list_display = ['email', 'first_name', 'last_name', 'is_active']
+    fieldsets = (
+        (None, {'fields': ('email', 'password',)}),
+        ('Personal info', {'fields': ('first_name', 'last_name', 'email', 'date_joined', 'last_login',)}),
+        ('Permissions', {'fields': ('is_active', 'is_staff', 'is_superuser', 'groups', 'user_permissions',)}),
+    )
+    add_fieldsets = (
+        (None, {'fields':
+                ('email', 'first_name', 'last_name', 'password1', 'password2'),
+                }),)
+    search_fields = ('email', 'first_name', 'last_name',)
+    ordering = ('last_name', 'first_name',)
+    actions = ['send_email_invitation']
+
+    def send_email_invitation(self, request, queryset):
+        for user in queryset:
+            if user.is_active is False:  # email doesn't send unless user is active
```

Comment:
@cwisecarver Basically I wanted to make sure that any accounts we sent invitations to would be inactive until a password reset. 
But certainly this does not cover the case where somebody had already reset their password but wanted to do so again. If they received the email and don't change their password, their account would be inactive even though it should be active. 
Perhaps we can circumvent this by adding a new confirmed flag to determine if a user has had an initial password reset?